vectorx/font/TextLine.hx

```diff
@@ -282,7 +290,7 @@ class TextLine
                 lastSpanInLine.updateString();
             }
 
-            if(lastSpanInLine.haveShadow() && lastSpanInLine.attachmentId == null)
+            if(lastSpanInLine.haveShadow() /* && lastSpanInLine.attachmentId == null*/)
```

Comment:
damn I forgot about that comment. actually I think the shadows are not even working for attachments. So it does actually work for the line width, just not for the shadows...